src/main/java/com/neatresults/mgnltweaks/json/JsonBuilder.java

```diff
@@ -668,4 +665,21 @@ public void setJson(String json) {
         this.preexisingJson = json;
     }
 
+    // TODO: Move to utils class, e.g. Java8Util
```

Comment:
`PropertyUtil.getValueString(p)` or just `getValueString(p)` as it is also already statically imported in the class ;)